resources/quality/blocks/Nozzle/blocks_global_0.20_standard.inst.cfg

```diff
@@ -0,0 +1,18 @@
+[general]
+version = 4
+name = Standard Quality
+definition = blocks_base
+
+[metadata]
+setting_version = 19

```

Comment:
setting version is 20 now (not a big deal if you don't fix it, but it will make the tests fail right now)